pages/storage_and_backup/object_storage/cold_archive_overview/guide.de-de.md

```diff
@@ -39,13 +45,15 @@ The service allows you to focus on building and deploying cloud applications whi
 
 The service is fully managed by OVHcloud and accessible through S3 API.
 
-**3 steps operation**
+**5 steps operation**
 
-1. Store your data in an Object Storage bucket first in RBX Region
-2. Archive it on tapes
-3. Restore your data or/and delete your archive
+1. Store your data in an Object Storage bucket first in RBX Region – this bucket is a temporary bucket,
```

Comment:
region name is "rbx-archive", there is another region named "rbx" but is not the same

Reply:
done